tests/devices/unit_tests/util/test_lookup_tables.py

```diff
@@ -0,0 +1,22 @@
+from pytest import mark
+
+from dodal.devices.util.lookup_tables import (
+    linear_interpolation_lut,
+)
+
+
```

Comment:
Should: Would be good to have a test in here for a non-monatomic file too

Reply:
test added